docs/basic_client.rst

```diff
@@ -44,7 +44,7 @@ HTTPBin RestConsumer
     class HTTPBinGetThenPost(object):
         def __init__(self, \*args, \**kwargs):
             super(HTTPBinGetThenPost, self).__init__(\*args, \**kwargs)
-            self._api = HTTPBinRestClient()
+            self._api = HTTPBinRestClient(timeout=60)
```

Comment:
Just showing the ability to pass in a custom timeout setting.. the docs still need work here, but right now I was just trying to get this in quickly.